### DISCUSSION TITLE
Anyone encountering like this?

### DISCUSSION THREAD

**Discussion by censonalvin** [GrapesJS/grapesjs#4595] [Q&A] [answered]
this happen yesterday after the release of the new version. 

any idea/s how to fix?

![image](https://user-images.githubusercontent.com/17976518/190536019-bff3543f-412a-4161-9f73-b8fc1cfdb84e.png)

![image](https://user-images.githubusercontent.com/17976518/190536173-8608907d-fe5d-4001-9aab-5f4c0b4cb727.png)

**Comment by ronaldohoch** [comment]
Can you show the code where are you using the "BlockManager"? Because i tryed in demo `editor.BlockManager` and it worked.

**Reply by censonalvin** [reply]
ohhh sorry i forgot to comment that i fixed it already. I just made my version back to 0.19.X. I  don't know what happen when i choose the new version.

**Comment by censonalvin** [comment, marked as answer]
I just change back to version 0.19.X of grapesJS to fix it.

(I don't know why it became like this when I choose the updated version)
